src/libc/vdp_vdu.c

```diff
@@ -374,3 +482,81 @@ void vdp_reset_sprites( void )
 	VDP_PUTS( vdu_sprite_reset );
 }
 
+/* Advanced buffered commands (selected) */
+
+static VDU_ADV_CMD_ui16 vdu_adv_write_block  = { 23, 0, 0xA0, 0xFA00, 0, 0};
+//static VDU_ADV_CMD      vdu_adv_call_buffer  = { 23, 0, 0xA0, 0xFA00, 1};
```

Comment:
I think this is a mistake and should be included?

Reply:
I did start to add it, but I had no way of testing it as I don't really get the "commands in a VDP buffer" thing.
So, I thought it best I leave it out.
Certainly, I could just add it verbatim from the docs and hope it is correct.

Reply:
Yes, it looks correct. Please add it.

Reply:
Updated the head of my main to remove the commented code.
I think probably I would need to create a new PR with all the related "command buffer" VDP commands (1, and 6 though 13) so that they can be used together properly.

That way, this request just adds the graphics advanced buffer commands.